src/edit/DistortableCollection.Edit.js

```diff
@@ -303,28 +303,28 @@ L.DistortableCollection.Edit = L.Handler.extend({
         statusUrl = opts.statusUrl + data;
         // repeatedly fetch the status.json
         self.updateInterval = setInterval(function intervalUpdater() {
-          $.ajax(statusUrl + '?' + Date.now(), {// bust cache with timestamp
-            type: 'GET',
-            crossDomain: true,
-          }).done(function(data) {
-            opts.updater(data);
-          });
+          var request = new Request(statusUrl+'?'+Date.now(), {method: 'GET'});
+          fetch(request).then(function(response) {
+            if (response.status === 200) {
```

Comment:
Hard coding this to 200 is not good. What if later it changes to 202, which is also valid code, just like any 200 status code. This should be replaced with `response.ok`. [Read More](https://developer.mozilla.org/en-US/docs/Web/API/Response/ok)